runtime/mount/kernel.py

```diff
@@ -842,7 +866,7 @@ async def send_output_value(
                     "type": "output_value",
                     **(id_fields),
                     **(key_fields),
-                    "plotly_json": res.to_json(),
+                    "plotly_json": json.dumps(serialize_plotly_figure(res)),
```

Comment:
del `json.dumps`?

Reply:
no, this is supposed to be a string apparently
`to_json` returns a string